lib/generators/migration/templates/migration_spec.rb.tt

```diff
@@ -0,0 +1,37 @@
+require_migration
+
+describe <%= spec_class_name %> do
+  # let(:<%= spec_symbol_name %>_stub) { migration_stub(:<%= spec_class_name %>) }
```

Comment:
I don't think this is right - it looks like the stub has to do with the migration name, but it is kind of unrelated - it's more about the models defined in the migration.  I think the old way with the hardcode MyModelStub looked clearer.